requirements.txt

```diff
@@ -1,4 +1,4 @@
-urllib3<1.24
-six==1.11.0
+urllib3<=1.24
+six<=1.11
```

Comment:
This won't fix my problem.  Why be so strict?  six uses semver.  Can you use `six>=1.11.0,<2.0` instead?

Reply:
```suggestion
six<=1.12.0
```

Reply:
This would free me up, too. I'm currently facing an unresolvable conflict in a modified version of mirumee/saleor, wherein graphql-relay requires `six>=1.12.0`

I can't build or ship. As noted, semver means it should be safe to `six>=1.11.0,<2.0`